### PR TITLE
feat!: updated to latest probot framework

### LINKED ISSUE #183: Service down
Hey,
First, thanks for this project, I know it's a one-man project and that can be quite exhausting.

It seems the service is down, since 1h I would say.
Is there anything we can do to help? ☺️ 

Best Regards,

### PR DESCRIPTION
**Pull request by mdvorak**
- updated tests
- use `@conventional-commits/parser` instead of `parse-commit-message` (it is no longer maintained) - this might change bot behavior in some corner cases

I've done some simple tests, looks like it works.

Note that for some reason, tests now take forever to execute (about 1s per test), but I have no idea why. But I still think it is better to test it with original `ProbotOctokit` and `Context`, instead of trying to emulate their behavior.

Should resolve #183 after being deployed.

### CODE REVIEW

**Comment by uzikilon** [comment]
Thanks for doing this, @mdvorak. I would love to get the bot running again on our repo/

File: .gitignore [hunk 1]
```diff
@@ -3,3 +3,5 @@
 coverage
 node_modules
 private-key.pem
+.idea/
```
Comment (pjurczynski) [inline .gitignore:6]: imho this should be a part of your own global gitignore rather than a part of the repository (a good reasoning behind it is written at https://blog.martinhujer.cz/dont-put-idea-vscode-directories-to-projects-gitignore/)

Comment (mdvorak) [reply .gitignore:6]: Thanks for the link, its really nice write-up. Honestly, I'm used to adding these everywhere, since its few extra words in one file, vs constant dealing with someone committing IDE files. I did not even notice I have it in the PR.

**Comment by stevideter** [comment]
Any chance of getting this merged in? We depend on this bot across several repos and this is really slowing us down.

**Comment by Shinigami92** [comment]
Feel free to take this https://github.com/faker-js/faker/commit/d94159f155cd58ba8ebfdbf0d2dd642a418175ab over into your projects

File: .github/dependabot.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "weekly"
+
+  - package-ecosystem: "npm"
```
Comment (MrRight101) [inline .github/dependabot.yml:9]: .github/dependabot.yml

**Review by MrRight101 — COMMENTED** [review]
Fast trans
